docs/kyma-environment-broker/03-09-subaccount-cleanup-cronjob.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Subaccount Cleanup CronJob
+type: Details
+---
+
+Each SKR instance in KEB database belongs to global account and subaccount.
+Subaccount Cleanup is an application which periodically calls CIS service about SUBACCOUNT_DELETE events 
+and based on those events trigger de-provision action on SKR instance which subaccount belongs to.
+
+## Details
+
+All Subaccount Cleanup process is divided into several steps
+
+1. Fetch SUBACCOUNT_DELETE events from CIS service
+
+    a. CIS client makes first call to CIS service, as a response it gets list of events  divided into pages, 
+       total number of events and total number of pages
```

Comment:
```suggestion

```

Reply:
I'd remove that line as it brings more confusion than explanation. Also, please separate each point with a blank line so that the list renders well.